# Hekateros hekateros_control/GotoParkedPos service

---
int8 rc   # return code
